metalsmith.json

```diff
@@ -33,14 +33,6 @@
       "source": "./assets",
       "dest": "."
     },
-    "metalsmith-sass": {
-      "outputStyle": "expanded",
-      "includePaths": [
-        "assets/vendor",
-        "assets/vendor/bootstrap-sass-twbs/assets/stylesheets",
-        "assets/vendor/fontawesome/scss/"
```

Comment:
As I mentioned before, metalsmith-sass doesnt pass all the arguments you pass to it to node-sass, but rather hard-codes the 2-3 the author thought was important.

Also the SCSS files absolutely don't belong in the metalsmith src directory, but we were doing it so we could use metalsmith-sass.

Ideally we would just use node-sass directly, but the CLI interface doesnt support multiple includePaths directories yet. 

Whereas this specific implementation has no real downside.